setup.py

```diff
@@ -113,7 +113,7 @@ def parse_cmdline_args(extra_args_msg):
     parser.add_argument('--sysimage', nargs=1, default=None,
                         help='Path to golden image to be coppied from either '\
                             ' local path or url. This will skip debootstrap stage!')
-    parser.add_argument('--vmd-cfg', default='golden.arm.vmd',
+    parser.add_argument('--vmd-cfg', default='arm.vmd',
```

Comment:
One thing I always do is look at "ls wherever" to see how all these template files "look and feel" in a directory listing.   Things like: if instead of <ARCH>.vmd, would the directory look better if they were "vmd.<ARCH>" because then all the vmds are visually grouped together?  Just a thought, not a blocker.